src/tokenizer.rs

```diff
@@ -807,6 +807,9 @@ pub struct Tokenizer<'a> {
     /// If true (the default), the tokenizer will un-escape literal
     /// SQL strings See [`Tokenizer::with_unescape`] for more details.
     unescape: bool,
+    /// If true, the tokenizer will not escape % and _, for use in in LIKE patterns. See
+    /// [`Dialect::ignores_like_wildcard_escapes`] for more details.
+    ignore_like_wildcard_escapes: bool,
```

Comment:
was it a reason to store this value here vs relying solely on the dialect via `self.dialect.ignores_like_wildcard_escapes()` when needed?